j2735_v2x_msgs/CMakeLists.txt

```diff
@@ -42,10 +43,29 @@ rosidl_generate_interfaces(
         std_msgs
 )
 
+install(DIRECTORY include/${PROJECT_NAME}/
+  DESTINATION include/${PROJECT_NAME}
+)
+
 ament_export_dependencies(rosidl_default_runtime)
+ament_export_include_directories(include)
 ament_package()
 
 install(
   FILES rule.yml
   DESTINATION share/${PROJECT_NAME})
 
+if (BUILD_TESTING)
+  enable_testing()
+
+  include(cmake/ament_auto_find_test_dependencies.cmake)
```

Comment:
Can we add a comment to these include statements similar to the comment we have in`motion_computation`? https://github.com/usdot-fhwa-stol/carma-platform/blob/d385568f9f10c695f547c9bf58be68cf89ea0058/motion_computation/dependencies.cmake#L21

```
# These CMake commands were added to ament_cmake_auto in ROS 2 Humble. Until
# CARMA supports ROS 2 Humble, we will use package-local copies.
```